interview-questions.md

```diff
@@ -8,33 +8,64 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is a language that allows for writing HTML code in javascript. a syntax difference between the two is that you need to use {} to indicate that you are writing javascript opposed to <></> for HTML
 
-Researched answer:
+Researched answer: (the source: https://www.geeksforgeeks.org/what-are-the-differences-between-jsx-and-html/)
+
+(source Language): JSX (JavaScript + XML) is an extension of JavaScript that permits you to write down HTML directly within JavaScript. which features a few benefits of creating your code more readable and exercising the complete power of JavaScript within HTML. JSX is in some ways almost like HTML, however, it does accompany certain conspicuous differences which we’ll cover within the next section. Since JSX isn’t a legitimate JS code, it must be compiled into JS with a tool like Babel or similar.
+
+(My Interpretation)
+JSX is an extension of javascript(I know the source said that) that filled a need to brige javascript and HTML for better functionality when programing. And like many things where there a benefits there are also shortcomings in the way of having to only return a single parent element. 
 
 1. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: Yarn is Yet Another Resource Negotiator. the files that are modified when runnin yarn are the node module files to add dependencies for related files and the yarn lock file that is created.
 
 Researched answer:
 
+(The Source: https://en.wikipedia.org/wiki/Yarn_(package_manager))
+
+(language from the source: Yarn is one of the main JavaScript package managers,developed in 2016 by Meta (formerly Facebook) for the Node.js JavaScript runtime environment. An alternative to the npm package manager, Yarn was created as a collaboration of Facebook, Exponent (now Expo.dev), Google, and Tilde (the company behind Ember.js) to solve consistency, security, and performance problems with large codebases.)
+
+(My Interpretation: Yarn as it's name states is Yet Another Resource negotiator. It is a package manager that will find, download and install files that you need for certain task. Looking into what yarn is ultimatily wasnt as frutifull in helping me understand as looking into what a package manager is. Because that is ultimatly what yarn is.)
```

Comment:
Here's a suggestion: Once you have gotten the definition, do another search to see what type of files and code snippets yarn adds to applications.